function-app/adb-to-purview/src/Function.Domain/Helpers/OlProcessing/OlMessageConsolodation.cs

```diff
@@ -228,11 +279,32 @@ private bool IsStartEventEnvironment(Event olEvent)
             return false;
         }
 
+        /// <summary>
+        /// Helper function to determine if the event is one of
+        /// the data source v2 ones which need to aggregate data
+        /// from the start and complete events
+        /// </summary>
+        private bool isDataSourceV2Event(Event olEvent) {
+            string[] special_cases = {"azurecosmos://", "iceberg://"}; // todo: make this configurable?
```

Comment:
Agreed, we should definitely make this part of the configuration instead.  Maybe something like openLineageDataSourceV2Prefixes as the parameter and have it separated by semicolon (;)

Similar to the [Spark_Entities config](https://github.com/microsoft/Purview-ADB-Lineage-Solution-Accelerator/blob/release/2.2/docs/configuration.md#experimental-app-settings).  Is there ever a case where "://" would not be part of the prefix?